### ISSUE TITLE
Include reference for glob operators

### ISSUE THREAD

**Issue by tnrich** [carrot/globtester#9]
Hey there,

It would be awesome if on the right side of the tool was a little directory of all the possible operators that you can use in the minimatch glob.

Kind of like how regex-101 gives some insight into the operators a given regex will take. 

Thanks!

**Comment by jescalan** [comment]
Hey Thomas!

I agree, that would be great. Want to take a stab at pull requesting it in?

**Comment by vvo** [comment]
Yes for this, but I don't even know what are the operators or a good list

**Comment by jescalan** [comment]
@vvo A couple resources:

- https://github.com/isaacs/minimatch#comparisons-to-other-fnmatchglob-implementations
- https://www.wikiwand.com/en/Glob_(programming)#/secton_Syntax

**Comment by vvo** [comment]
thx
